packages/app/src/cli/models/app/app.ts

```diff
@@ -174,7 +96,7 @@ export function isLegacyAppSchema(item: AppConfiguration): item is LegacyAppConf
  */
 export function isCurrentAppSchema(item: AppConfiguration): item is CurrentAppConfiguration {
   const {path, ...rest} = item
-  return isType(AppSchema, rest)
+  return isType(AppSchema.nonstrict(), rest)
 }
```

Comment:
The problem is that new apps created using the command `create-app` pulls the content of the templates that right now contains a legacy `toml` schema ([remix](https://github.com/Shopify/shopify-app-template-remix/blob/main/shopify.app.toml), [only extension](https://github.com/Shopify/shopify-app-template-none/blob/main/shopify.app.toml) )
To be honest it would be a great idea to remove the support for the `legacy` schema, because there are lots of castings and `if...else...` sentences checking this temporary legacy schema.
The problem is that the current schema requires a `name` and a `client_id`. Maybe we could make only the name required and fill it in with the name used to create the app.
Anyway, I would address this refactor in the final polishing PRs of this project